examples/hybrid/sphere/hadley_circulation.jl

```diff
@@ -0,0 +1,334 @@
+    norm(y0.ρq1 ./ ρ_ref.(coords.z))
+initial_mass = sum(y0.ρq1 ./ ρ_ref.(coords.z))
+mass = sum(sol.u[end].ρq1 ./ ρ_ref.(coords.z))
+rel_mass_err = norm((mass - initial_mass) / initial_mass)
+@test q1_error ≈ 0.0 atol = 0.05
+@test rel_mass_err ≈ 0.0 atol = 3e-6
+
+Plots.png(
+    Plots.plot(
+        sol.u[trunc(Int, end / 2)].ρq1 ./ ρ_ref.(coords.z),
+        level = 11,
+        clim = (-0.1, 3.5),
+    ),
+    joinpath(path, "q1_half_day_level_11.png"),
+)
+
+# Remapping infrastructure to have lat-altitude Plots
+Nq = Spaces.Quadratures.degrees_of_freedom(
+    Spaces.quadrature_style(hv_center_space),
+)
+remap_tmpdir = path * "/remaptmp/"
+mkpath(remap_tmpdir)
+datafile_cc = remap_tmpdir * "cc.nc"
+NCDataset(datafile_cc, "c") do nc
+    # defines the appropriate dimensions and variables for a space coordinate
+    def_space_coord(nc, hv_center_space, type = "cgll")
+    def_space_coord(nc, hv_face_space, type = "cgll")
+    # defines the appropriate dimensions and variables for a time coordinate (by default, unlimited size)
+    nc_time = def_time_coord(nc)
+
+    # define variables
+    nc_rho = defVar(nc, "rho", Float64, hv_center_space, ("time",))
+    nc_q1 = defVar(nc, "q1", Float64, hv_center_space, ("time",))
+
+    # write data to netcdf file
+    for i in 1:length(sol.u)
+        nc_time[i] = sol.t[i]
+
+        # write fields to file
+        nc_rho[:, i] = sol.u[i].ρ
+        nc_q1[:, i] = sol.u[i].ρq1 ./ sol.u[i].ρ
+    end
+    close(nc)
+end
+
+# write out our cubed sphere mesh
+meshfile_cc = remap_tmpdir * "mesh_cubedsphere.g"
+write_exodus(meshfile_cc, hv_center_space.horizontal_space.topology)
+
+# write out RLL mesh
+nlat = 90
+nlon = 180
+meshfile_rll = remap_tmpdir * "mesh_rll.g"
+rll_mesh(meshfile_rll; nlat = nlat, nlon = nlon)
+
+# construct overlap mesh
+meshfile_overlap = remap_tmpdir * "mesh_overlap.g"
+overlap_mesh(meshfile_overlap, meshfile_cc, meshfile_rll)
+
+# construct remap weight file
+weightfile = remap_tmpdir * "remap_weights.nc"
+remap_weights(
+    weightfile,
+    meshfile_cc,
+    meshfile_rll,
+    meshfile_overlap;
+    in_type = "cgll",
+    in_np = Spaces.Quadratures.degrees_of_freedom(
+        Spaces.quadrature_style(hv_center_space),
+    ),
+)
+
+# apply remap
+datafile_rll = remap_tmpdir * "data_rll.nc"
+apply_remap(datafile_rll, datafile_cc, weightfile, ["rho", "q1"])
+
+# load remapped data and create statistics for plots
+nt = NCDataset(datafile_rll, "r") do nc
+    lat = nc["lat"][:]
+    z = nc["z"][:]
+    time = nc["time"][:]
+    q1 = nc["q1"][:]
+    (; lat, z, time, q1)
+end
+(; lat, z, time, q1) = nt
+
+# calculate zonal average for plotting
+calc_zonalave(x) = dropdims(mean(x, dims = 1), dims = 1)
+
+q1_zonalave = calc_zonalave(q1)
+
+times = 0.0:dt:T
+anim = Plots.@animate for i in 1:length(times)
+    Plots.contourf(
+        lat,
+        z,
+        q1_zonalave[:, :, i]',
+        color = :balance,
+        linewidth = 0,
+        yaxis = :log,
```

Comment:
```suggestion
```
Why log here?

Reply:
Maybe this is the issue? (the output may be very distorted from this)

Reply:
That's a good question! I was referring to the way similar plots are done in ClimaAtmos, see this as an [example](https://github.com/CliMA/ClimaAtmos.jl/blob/873467ccadd6ef6fb85f4f122425dd9361a28649/examples/hybrid/define_post_processing.jl#L957)

Reply:
Yeah, makes sense. I'm testing it without now